model_api/python/openvino/model_api/models/classification.py

```diff
@@ -85,13 +84,13 @@ def parameters(cls):
 
     def postprocess(self, outputs, meta):
         outputs = outputs[self.out_layer_name].squeeze()
+        if not np.isclose(np.sum(outputs), 1.0, atol=0.01):
```

Comment:
Do you think that this is the right check that reflects what happens in the C++ part?

Reply:
C++ embeds softmax into `ov::Model`. We used to have it in Python as well. But we had to remove it to support `OVMSAdapter` in python which doesn't allow to manipulate `ov::Model` on the server side.